### ISSUE TITLE
Bitbucket Pipelines Codecov submission no longer works

### ISSUE THREAD

**Issue by platawiec** [JuliaCI/Coverage.jl#159]
I previously used Coverage.jl in Bitbucket pipelines without any issues. With the newest update to LibGit2, I am getting an error. I have a private Docker container with all my dependencies. The build script looks like:

```
# Bitbucket portions
umask 000
git clone --branch="master" --depth 50 https://x-token-auth:$REPOSITORY_OAUTH_ACCESS_TOKEN@bitbucket.org/MYREPO.jl.git $BUILD_DIR ; git reset --hard ### ; git remote set-url origin git@bitbucket.org:MYREPO.git
chmod 777 $BUILD_DIR
# my commands, worked previously
mkdir ~/.julia/v0.6/MYPKG && cp -r ./ ~/.julia/v0.6/MYPKG
julia -e 'Pkg.clone(pwd()); Pkg.build("MYPKG"); Pkg.test("MYPKG"; coverage=true)'
julia -e 'cd(Pkg.dir("MYPKG")); using Coverage; Codecov.submit_local(Codecov.process_folder())'
```
The testing and file coverage generation goes fine. Where previously it would submit the report, I now get:
```
ERROR: GitError(Code:ENOTFOUND, Class:Repository, Could not find repository from '/root/.julia/v0.6/MYPKG/..')
Stacktrace:
 [1] macro expansion at ./libgit2/error.jl:99 [inlined]
 [2] Base.LibGit2.GitRepo(::String) at ./libgit2/repository.jl:10
 [3] #submit_local#11(::Array{Any,1}, ::Function, ::Array{Coverage.FileCoverage,1}, ::String) at /root/.julia/v0.6/Coverage/src/codecovio.jl:170
 [4] submit_local(::Array{Coverage.FileCoverage,1}) at /root/.julia/v0.6/Coverage/src/codecovio.jl:170
```

I will investigate this and update if I can find a way to set up the build script in a way that works. Any hints or help is appreciated.

**Comment by platawiec** [comment]
I think I've found the issue. The docs mention:

https://github.com/JuliaCI/Coverage.jl/blob/466bccda85dc994f5b34e7d8d86e1635fedbaf69/src/codecovio.jl#L161-L168

But the actual function for `submit_local` is:
https://github.com/JuliaCI/Coverage.jl/blob/466bccda85dc994f5b34e7d8d86e1635fedbaf69/src/codecovio.jl#L169

Namely, `dir` is set to `joinpath(pwd(),"..")`, not `pwd()` as the documentation strictly indicates.

I changed my build command to:
```
julia -e 'cd(Pkg.dir("MYPKG")); using Coverage; Codecov.submit_local(Codecov.process_folder(), pwd())'
````

And was able to build successfully.
